third_party/aog-payjs/README.amp

```diff
@@ -0,0 +1,9 @@
+URL: https://github.com/google/payjs
+License: Apache 2.0
+License File: https://github.com/google/payjs/blob/master/LICENSE
+
+Description:
+Copy of Google Pay integration library
+
+Local Modifications:
+Removed bazel build files
```

Comment:
Can you add instructions here on how to generate the bundled aog-payjs/payjs.js from the google/payjs library?

Reply:
done